LICENSE-APACHE

```diff
@@ -1,13 +1,176 @@
-Copyright 2022 MapLibre contributors
```

Comment:
@nyurik , is it possible to add back the "Copyright 2022 MapLibre contributors" and have github recognize apache like to does MIT?